.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: feature-request
+assignees: ''
+
+---
+
+<!--
+If your issue is a usage question, please submit it in one of these other
```

Comment:
Do we want to mention `fairlearn-proposals` here? I'm also good leaving as is for now. We can move the feature requests into `fairlearn-proposals` repo as needed.

Reply:
I think people may get confused if we give them the option. Better to triage that ourselves.

Reply:
sounds good.